tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -163,4 +163,77 @@ public function userCreatesAPublicLinkShareWithSettings(string $user, TableNode
 			)
 		);
 	}
+
+	/**
+	 * @When /^user "([^"]*)" has created the following link share:$/
+	 *
+	 * @param string $user
+	 * @param TableNode|null $body
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function userHasCreatedALinkShareWithSettings(string $user, TableNode  $body):void {
+		$this->userCreatesAPublicLinkShareWithSettings($user, $body);
+	}
+
+	/**
+	 * @When /^user "([^"]*)" updates the last public link share using the Graph API with$/
+	 *
+	 * @param string $user
+	 * @param TableNode|null $body
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function userUpdatesLastPublicLinkShareWithSettings(string $user, TableNode  $body):void {
+		if (!isset($this->featureContext->getJsonDecodedResponse()['id'])) {
+			throw new Error('Response did not contain share id ' . $this->featureContext->getJsonDecodedResponse()['id'] . ' for the created public link');
+		}
```

Comment:
it's better to save last created link share data into separate store instead of using `$this->response`